[*] running method core_negotiate_tlv_encryption
[*] Negotiating TLV encryption
[*] RSA key: 30820122300d06092a864886f70d01010105000382010f003082010a0282010100b6b44fc6c4aa07162059a92a612e27d48407898d87022672ca9f7cb43226499f2985996a3f035f113a90b67eb2c891f2028bebd98fc9f7d69348354a7add489f9a687321ba708782d358b8cb85a0304c1ea90e46e7e3e61810821bfb4c0ee546d3a6192fa1f54b300577b81402851b9746b7439ae8d3cc89b102905ccf2c1b1ddbb43c68d9be671bbdba4ac5fb22ea672413c0ccb9a1291ab73b01cc89adfc9c6a7db58c1a527dcb1c7a70787d22583bfff8e8f41f4cd8bee2b7e6963ebff99f3bc2efb3e4fafc215d627dbf83e0b5411bc9b8933685224515e7a4a7e1d5413f59536d6c7b018744a90e3449e2fe10ebfecf70083ad13e8ca64b4e77364cf5790203010001
[*] AES key: 0x1a27322749f14008e4e3bdce9c5d37887ea6ff92b6b8e816ef11c07e7fd2c720
[*] Encrypted AES key: 0x8f157035260076da456d35ef8939d57994c0a42dd74625448a11054e0cb904d33c1cd8495a59d7413e4da77d793d643ad549fa17130e8f0fb36b8aa0f27736e551f7354af880f6dbe8e883e69f141a77cc7f828c08c5cac97080c2b1e5f52e184a8fa2118d0478c80508dcd6d7471df191f0d87fd6507288fef25a58753698b239e10ef785430f2226559c665b35a5f7919d08ba2fc91eec87ee6eeaa364fb87be0ba7a1cf0b8538efea790f84aa2e6900a6e1f8b0021ce49057dc321a0245d73d47e3396f3d45cbf547bec7b79baf03d822a61619fb0fb9aa9771b4e7b889d955acb124c6920be1203651b7a84a57f26ff907b5e2895866ec68350d2688c6ee
[*] TLV encryption sorted
[*] sending response packet
[*] running method core_set_session_guid
[*] sending response packet
[*] running method core_enumextcmd
[*] sending response packet
[*] running method core_enumextcmd
[*] sending response packet
[*] running method core_loadlib
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_sys_config_getuid
[*] sending response packet
[*] running method stdapi_sys_config_sysinfo
[*] sending response packet
[*] running method core_set_uuid
[*] sending response packet
[*] running method stdapi_net_config_get_interfaces
[*] sending response packet
[*] running method stdapi_net_config_get_routes
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_separator
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir-gxoqzinb'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 1 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 1
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 2 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 2
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 3 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 3
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 4 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 4
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 5 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 5
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\boot.ini'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\pagefile.sys'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '/etc/master.passwd'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\notepad.exe'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\calc.exe'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 6 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 6
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 7 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 7
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 8 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 8
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 9 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 9
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 10 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 10
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 11 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 11
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 12 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 12
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

[*] sending response packet
[*] running method stdapi_fs_file_move
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 13 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 13
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir/directory'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 14 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 14
[*] sending response packet
[*] running method stdapi_fs_ls
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'command -v ln || which ln && echo DIELRJAN']
[*] added process id: 3227, handle: 1
[*] added channel id: 15 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_read
[-] method core_channel_read resulted in error: #1
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 1
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 16 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 16
[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir $(pwd)/meterpreter-test-dir.lnk']
[*] added process id: 3232, handle: 2
[*] added channel id: 17 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 2
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.lnk'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.1'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.2'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 18 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 18
[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir.1 $(pwd)/meterpreter-test-dir.2/link']
[*] added process id: 3238, handle: 3
[*] added channel id: 19 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 3
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
